invcache/invalidating_cache.go

```diff
@@ -0,0 +1,165 @@
+package invcache
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/goware/cachestore"
+	"github.com/goware/pubsub"
+)
+
+const (
+	ChannelID = "cache_invalidation"
```

Comment:
lets rename this to "DefaultChannelID" .. as others may want to use their own channelID value

Reply:
Done!